tests/event.rs

```diff
@@ -2,12 +2,19 @@ use compio::event::Event;
 
 #[test]
 fn event_handle() {
-    compio::task::block_on(async {
-        let event = Event::new().unwrap();
+    let event = Event::new().unwrap();
+
+    std::thread::scope(|scope| {
         let handle = event.handle();
-        std::thread::scope(|scope| {
-            scope.spawn(|| handle.notify().unwrap());
+        let wait = event.wait();
+        scope.spawn(move || {
+            std::thread::sleep(std::time::Duration::from_secs(1));
+            handle.notify().unwrap()
+        });
+        scope.spawn(move || {
```

Comment:
What's the reason of this change?

Reply:
mio won't receive event before register (edge triggered) so i was trying to fix event but apparently this is still not working. Its still wip.

Reply:
That will make `Event` on windows cannot find the current runtime.